templates/partials/summary/summary.html

```diff
@@ -1,24 +1,22 @@
 {%- if content.summary.collapsible -%}
-  {%- include 'partials/summary/collapsible-summary.html' -%}
+    {%- include 'partials/summary/collapsible-summary.html' -%}
 {%- else -%}
-  {% from "components/summary/_macro.njk" import onsSummary %}
+    {% from "components/summary/_macro.njk" import onsSummary %}
 
-  {% set summary_sections = [] %}
-  {% set summary_type = content.summary.summary_type %}
-  {% set view_submitted_response = content.summary.view_submitted_response %}
-
-
-  {%- for section in content.summary.sections -%}
-    {% set summary_groups = [] %}
-    {%- if section.groups -%}
-      {%- for group in section.groups -%}
-        {%- if group.blocks -%}
-          {% do summary_groups.append
-            (
-              {
-                  "groupTitle": group.title if group.title else None,
-                  "id": group.id if group.id else None,
-                  "rows": map_summary_item_config(
+    {% set summary_sections = [] %}
+    {% set summary_type = content.summary.summary_type %}
+    {% set view_submitted_response = content.summary.view_submitted_response %}
+    {%- for section in content.summary.sections -%}
+        {% set summary_groups = [] %}
+        {%- if section.groups -%}
+            {%- for group in section.groups -%}
+                {%- if group.blocks -%}
+                    {% do summary_groups.append
+                    (
+                    {
+                    "groupTitle": group.title if group.title else None,
+                    "id": group.id if group.id else None,
+                    "rows": map_summary_item_config(
```

Comment:
This do has been flattened, is this another place where we need to trun djlint off?

Reply:
Yeah, had to expand the djlint off exception in this block and include the `for` loop.